app/views/content/service_privacy_notice.md

```diff
@@ -125,13 +126,14 @@ We may contact you to ask if you would like to participate in user research. Thi
 [Opt out of being contacted for research](https://forms.gle/dkZtK1g8ipKTD3s66).
 
 ### Getting insight to make government policy better
-We’ll analyse your data to help us inform government policy around teacher recruitment and retention. When we do, we’ll anonymise your personal information so you cannot be identified.
+We’ll analyse your data to help us inform government policy around teacher recruitment and retention. If you use other digital services for which the Data Controller is also the Department for Education (for example, Find Postgraduate Teacher Training, or Get Into Teaching services), we may use your IP address to understand how you are using these services. This allows us to ensure that public funds are being spent effectively.
 
 ### How long we keep your data
 
 | **Usage** | **Retention period** |
 | --- | --- |
 | Candidate application form | 7 years |
+| Analytics data in data warehouse | 7 years |
```

Comment:
"Data warehouse" is jargon, let's be specific.

```suggestion
| Analytics data in Google BigQuery | 7 years |
```

Reply:
not entirely convinced people are more likely to understand "Google BigQuery" than "data warehouse"

Reply:
Me either - resolving this for now.

Reply:
Fair point! 😄  Do we need to name the provider (Google) for any reason? We've got "Google Analytics" and "Zendesk Support Desk" below. Should we also make some attempt to explain what "analytics data" refers to?

Reply:
I'm actually more convinced on your point now, I'll add Google to it.